apps/nextra/pages/en/build/sdks/ts-sdk.mdx

```diff
@@ -6,7 +6,7 @@ The TypeScript SDK allows you to connect, explore, and interact on the Aptos blo
 - [Source Code](https://github.com/aptos-labs/aptos-ts-sdk)
 - [Reference Docs](https://aptos-labs.github.io/aptos-ts-sdk/@aptos-labs/ts-sdk-latest)
 
-# Quickstart
+## Typescript Quickstart
```

Comment:
```suggestion
## TypeScript Quick Start
```

Reply:
Haha I think it's one word actually, like https://cloud.google.com/shell/docs/quickstart

Reply:
At least `Typescript` -> `TypeScript`

Reply:
You got it